spec/support/a_framework_assessment.rb

```diff
@@ -0,0 +1,712 @@
+RSpec.shared_examples 'a framework assessment' do |assessment_type|
```

Comment:
This approach seems fine, only minor concern is that it uses a mixture throughout of both `assessment_type` (e.g. `:person_escort_record`) and `described_class`. I think I'd prefer to only see one of those used, with a slight preference towards `assessment_type` which would allow this to be potentially included in other specs. Admittedly that's unlikely to happen but it would mean this reads more cleanly too I think.

```suggestion
RSpec.shared_examples 'a framework assessment' do |assessment_type|
  subject { create(assessment_type) }
  
  let(:assessment_class) { assessment_type.to_s.classify }
  
  ...
```

..and then use `assessment_class` rather than `described_class` throughout. Thoughts?